s3cortxutils/s3confstore/s3confstore/cortx_s3_confstore.py

```diff
@@ -33,7 +33,11 @@ def __init__(self):
 
   def load_config(self, index: str, backend_url: str):
     """Load Config into constore"""
-    Conf.load(index, backend_url)
+    if not backend_url.strip():
+      print("backend_url is empty")
```

Comment:
can this be more verbose, and help in debugging issue problem. print as much info or any variable values as possible.

Reply:
I will ad more information.